makefile

```diff
@@ -9,7 +9,9 @@ node_modules:
 
 pretty: node_modules
 	$(PRETTIER) --write src/main/webapp/*.{html,css}
+
 	find src/main/java -iname *.java | xargs $(CLANG_FORMAT) -i
+	find src/test -iname *.java | xargs $(CLANG_FORMAT) -i
 	find src/main/webapp -iname *.js | xargs $(CLANG_FORMAT) -i
```

Comment:
BTW in `master` I reconfigured the linter and makefile to test all java/html/js/css files in all directories.  Just be sure to not overwrite those changes when merging.  If you want you can manually move the new configs over (.github/, makefile)